shell/testing/tester_main.cc

```diff
@@ -31,6 +32,48 @@
 
 namespace flutter {
 
+class TesterPlatformView : public PlatformView,
+                           public GPUSurfaceSoftwareDelegate {
+ public:
+  TesterPlatformView(Delegate& delegate, TaskRunners task_runners)
+      : PlatformView(delegate, std::move(task_runners)) {}
+
+  std::unique_ptr<Surface> CreateRenderingSurface() override {
```

Comment:
Nit: Please specify the interface that contains the definition being overridden. So, `|PlatformView|` I believe in this case.

Reply:
Done